Code/tests/src/network.cpp

```diff
@@ -1,331 +1,331 @@
-#include "catch.hpp"

```

Comment:
Did you change the line endings by any chance ?

Reply:
Good catch. I had autocrlf = input in my linux .gitconfig
Won't happen again!